job-executor/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+	    <groupId>uk.gov.justice.services</groupId>
+        <artifactId>job-manager</artifactId>
+	    <version>0.0.1-SNAPSHOT</version>
+    </parent>
+    
+	<modelVersion>4.0.0</modelVersion>
+
+    <artifactId>job-executor</artifactId>
+    <properties>
+        <cpp.service-component>EVENT_PROCESSOR</cpp.service-component>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>javax</groupId>
+            <artifactId>javaee-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+		<!--
+        <dependency>
+            <groupId>uk.gov.moj.cpp.common</groupId>
+            <artifactId>event-processor</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.services</groupId>
+            <artifactId>event-source</artifactId>
+        </dependency>
+		-->
+        <dependency>
+            <groupId>uk.gov.justice.services</groupId>
+            <artifactId>jobstore-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.services</groupId>
+            <artifactId>jobstore-persistence</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <!--Test-->
+        <dependency>
+            <groupId>uk.gov.justice.utils</groupId>
+            <artifactId>test-utils-logging-simple</artifactId>
+            <type>pom</type>
+            <scope>test</scope>
+			<version>1.16.0</version>
```

Comment:
version should be set in the parent pom as a property

Reply:
Done